src/components/SemesterView.vue

```diff
@@ -196,6 +196,9 @@ export default {
         }
       }
       this.openSemesterConfirmationModal(type, year, false);
+
+      // Update requirements menu from dashboard
+      this.$parent.updateRequirementsMenu();
```

Comment:
Did not know we had so many $parent references still. Making an item on Monday for myself

Reply:
Very bad practice and I'm really lazy. I'll fix this case, but we do have many $parent references still indeed

Reply:
Yeah its essentially just duplicating existing code so fine here, but I still need to resolve.